build.gradle

```diff
@@ -121,11 +121,13 @@ allprojects {
     if ("$System.env.JENKINS_URL" == 'https://jenkins.pegasys.tech/') {
       maven { url "https://nexus.int.pegasys.tech/repository/consensys-pegasys/" }
       maven { url "https://nexus.int.pegasys.tech/repository/jcenter/" }
+      maven { url "https://repository.apache.org/content/repositories/snapshots/" }
```

Comment:
Is adding the apache snapshot repo really needed?  I was under the impression Tuweni was on released versions now.

Reply:
Yes, absolutely. I would recommend you only depend on official releases - the snapshot repository apparently can blacklist folks if they pull too much from it.

Reply:
Release vote is now ongoing: https://t.co/o9Z2aO6OJ9?amp=1

Reply:
> I would recommend you only depend on official releases 

+1